lib/chai/core/assertions.js

```diff
@@ -1500,6 +1505,85 @@ module.exports = function (chai, _) {
   Assertion.addMethod('keys', assertKeys);
   Assertion.addMethod('key', assertKeys);
 
+  function assertStrings (strings) {
+    _.expectTypes(this, ['string']);
+    var obj = flag(this, 'object')
+      , negate = flag(this, 'negate')
+      , stringsType = _.type(strings)
+      , str
+      , deepStr = ''
+      , ok = true
+      , mixedArgsMsg = 'when testing strings against strings you must give a single Array|String argument or multiple String arguments';
+
+    if (stringsType === 'Array') {
+      if (arguments.length > 1) throw new Error(mixedArgsMsg);
+    } else {
+      strings = Array.prototype.slice.call(arguments)
+    }
+
+    if (strings.length === 1) {
+      return this.string(strings[0])
+    }
+
+    if (!strings.length) throw new Error('strings required');
+
+    var len = strings.length
+      , any = flag(this, 'any')
+      , all = flag(this, 'all')
+      , expected = strings;
+
+    if (!any && !all) {
+      all = true;
+    }
+
+    function containsStr (expectedStr) {
+      return ~obj.indexOf(expectedStr)
```

Comment:
Not a big fan of bitwise operators.
I prefer the following: 
```javascript
 return obj.indexOf(expectedStr) !== -1
```
I understand that we are using this kind of "trick" everywhere in the codebase, so I can live with that.

Also you're missing a semicolon here.